OrbitService/Utils.cpp

```diff
@@ -26,13 +28,14 @@
 
 namespace orbit_service::utils {
 
+namespace fs = std::filesystem;
+
 using ::ElfUtils::ElfFile;
 using orbit_grpc_protos::ModuleInfo;
 using orbit_grpc_protos::TracepointInfo;
 
 namespace {
 
-namespace fs = std::filesystem;
 const char* kLinuxTracingEvents = "/sys/kernel/debug/tracing/events/";
 
 ErrorMessageOr<uint64_t> FileSize(const std::string& file_path) {
```

Comment:
As daniel said - I would still modify this code to use error_code variants for std::filesystem